src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
+.CommentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #eee;
```

Comment:
Consider using a variable for the background color to improve maintainability and consistency in your styles. E.g., `$bg-color: #eee;`

Reply:
initiated `$_` var although this action caused figuring out about project structuring ...
ok